src/NJsonSchema.Tests/Generation/ArrayGenerationTests.cs

```diff
@@ -29,6 +29,27 @@ public async Task When_property_is_JArray_then_schema_with_any_array_is_generate
             Assert.True(arrayProperty.Item.ActualTypeSchema.IsAnyType);
         }
 
+#nullable enable
+        public class ClassWithArrayOfNullable
+        {
+            public string?[] Array { get; set; } = new string?[0];
```

Comment:
Can we add a test for `List<string?>` now that it's supported by Namotion.Reflection ?

Reply:
Added. Build fails in CI but local it's fine... investigating...